initial-checks.sh

```diff
@@ -11,9 +11,21 @@ uname -s | grep -i linux
 check_for_failure "Please run this script on Linux"
 
 if [ "$EUID" -ne 0 ];then
-  check_for_failure "Please run as root or with sudo"
+  echo "Please run as root or with sudo"
+  exit 1
```

Comment:
check_for_failure is a bash function I wrote that does exactly what lines 14/15 do:
https://github.com/ecohealthalliance/bsve-integration/blob/master/initial-checks.sh#L3-L8